src/cli/utils/output.ts

```diff
@@ -0,0 +1,25 @@
+import { writeFile } from 'fs';
+import { promisify } from 'util';
```

Comment:
It feels a little funny to make these generic functions when they are related to the lint command only. They had generic names because they didn’t need to be called formatLintOutput. 

Could we either put them back, or instead putting these in a util/ folder we put them in lint/?

Reply:
oh yeah, sorry I had some code before that used this and I extracted it :P I'm going to move it back.